blockdev_test.go

```diff
@@ -6,7 +6,7 @@ var (
 	deviceBlockString         = "-drive file=/var/lib/vm.img,id=hd0,if=none,format=qcow2,aio=threads,cache=unsafe,discard=unmap,detect-zeroes=unmap,readonly=on -device virtio-blk-pci,drive=hd0,serial=abc-123,disable-modern=true,addr=0x03,bus=pcie.0,logical_block_size=4096,physical_block_size=4096,scsi=off,config-wce=off,romfile=efi-virtio.rom,share-rw=on"
 	deviceBlockAddrString     = "-drive file=/var/lib/vm.img,id=hd0,if=none,format=qcow2 -device virtio-blk-pci,drive=hd0,serial=hd0,disable-modern=false,addr=0x07,bus=pcie.0,scsi=off,config-wce=off"
 	deviceBlockPFlashROString = "-drive file=/usr/share/OVMF/OVMF_CODE.fd,id=pflash0,if=pflash,format=raw,readonly=on"
-	deviceBlockPFlashRWString = "-drive file=uefi_nvram.fd,id=pflash1,if=pflash,format=raw"
+	deviceBlockPFlashRWString = "-drive file=uefi-nvram.fd,id=pflash1,if=pflash,format=raw"
```

Comment:
Let's leave these as they are; it's an unrelated change to enabling tpm on arm support